libs/core/likwid/include/hpx/likwid/likwid_init.hpp

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) 2022 Srinivas Yadav
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <likwid.h>
+#include <hpx/modules/runtime_local.hpp>
```

Comment:
Please #include the HPX headers first.  Also, shouldn't we wrap all of this header's content in a `#if defined(HPX_HAVE_LIKWID)`/`#endif`?